### ISSUE TITLE
Importing MBOX files always results in new folder (good) with only 1 mail in it (bad)

### ISSUE THREAD

**Issue by Cardboardgundam** [thundernest/import-export-tools-ng#45]
I tried importing a bunch of mbox files I made to migrate from Windows to Linux and always get the correct folder but there's only a single mail in it, the rest seems to get ignored. When I open the mbox files with Kate they look just fine.

Since I noticed there were some issues with file names I tried one renamed to 0.mbox, same result.

The mbox files were created with Becky! 2.74.02 on Windows 7 64.
I use Thunderbird 68.2.1 on Kubuntu 19.10 with import-export-tools-ng 4.0.4 to import.

**Comment by cleidigh** [comment]
@Cardboardgundam 
Have you tried importing a single mbox file successfully?

**Comment by Cardboardgundam** [comment]
@cleidigh
Nope, not one success. Got the most important recent messages as eml.

**Comment by cleidigh** [comment]
okay, if we could not import a single basic mbox file, something very basic must be at issue.
First, if you can try a test file I have the repository we will at least be working with the same file.
Second, if you could possibly create a simple mbox file that should work but does not and send that to me again, I will attempt to reproduce the problem.  I do not currently have a Linux set up as it's very difficult for me to operate as I need to use dictation.
@cleidigh

**Comment by Cardboardgundam** [comment]
Something weird happened.

I made a new small mbox with three messages and tried that and your test mbox file.
The test file had http code in it and got me an empty folder, the new small mbox was imported without error. Afterwards I created some more mbox files on the windows machine, those worked just fine. I now got everything imported on the linux machine, not one error.

It seems the not working mbox files were damaged somehow, I can only guess it was some software problem on the windows machine since I don't have those files anymore. Anyway, thanks for your help and this great tool.
